pkg/apprepo/apprepos_handler.go

```diff
@@ -310,6 +341,12 @@ func secretNameForRepo(repoName string) string {
 	return fmt.Sprintf("apprepo-%s-secrets", repoName)
 }
 
+// kubeappsSecretNameForRepo returns a name suitable for recording a copy of
+// a per-namespace repository secret in the kubeapps namespace.
+func kubeappsSecretNameForRepo(repoName, namespace string) string {
```

Comment:
isn't it better to reuse the method `secretNameForRepo`? To avoid confusion between similar methods.

Reply:
It is re-using `secretNameForRepo` in its implementation. I could have done this inline, which is perhaps what you mean? The only reason I've pulled it out to a function is because I'll need to generate the same secret name when deleting in a following PR, so rather than have the same inline code to create the kubeapps secret name, I've put it in a fn. Let me know if I missed something?

Reply:
It's just a bit confusing to know (if I were developing here) which method to use. For me, it'd be better to extend `secretNameForRepo(name, namespace)` allowing to set the namespace empty and adding a comment saying that the namespace is only needed if the secret is meant for the `kubeapps` namespace.